test/conformance/ptp/ptp.go

```diff
@@ -355,7 +348,7 @@ var _ = Describe("[ptp]", func() {
 						ptpMasterRunningPods = append(ptpMasterRunningPods, pod)
 					}
 				}
-				if discovery.Enabled() {
+				if discovery.Enabled() || isSingleNode {
```

Comment:
when it is singNode , you will label node as ptp/slave only

Reply:
fixed